flang/test/Lower/OpenMP/omp-seqloop.f90

```diff
@@ -21,3 +21,72 @@ program wsloop
 !FIRDialect:  }
         !$OMP END PARALLEL
 end
+
+!FIRDialect: func @_QPsub1() {
+subroutine sub1
+!FIRDialect:   {{.*}} = fir.alloca i32 {{{.*}}uniq_name = "_QFsub1Ei"}
+  integer :: i
+  integer :: arr(10)
+!FIRDialect:   omp.parallel {
+!FIRDialect:     {{.*}} = fir.alloca i32 {uniq_name = "i"}
+  !$OMP PARALLEL
+  do i=1, 10
+    arr(i) = i
+  end do
+!FIRDialect:     omp.terminator
+!FIRDialect:   }
+  !$OMP END PARALLEL
+end subroutine
+
+!FIRDialect: func @_QPsub2() {
+subroutine sub2
+!FIRDialect:   %1 = fir.alloca i32 {bindc_name = "i", uniq_name = "_QFsub2Ei"}
```

Comment:
We typically try to avoid matching ssa-values like `%1` since it doesn't seem robust.

Reply:
Done.